### ISSUE TITLE
Add .scalex() and .scaley() (feature request)

### ISSUE THREAD

**Issue by pozitron57** [cdelker/schemdraw#25]
Consider the Inductor2 element:
![image](https://github.com/cdelker/schemdraw/assets/9392655/97e548a4-fb5a-454b-8b18-c336cd39b206)

I propose to add `.scalex()` method to stretch the element in x direction like so:
![image](https://github.com/cdelker/schemdraw/assets/9392655/18a2b22c-41bb-49e5-bf72-1eb7b96c7960)

and `.scaley()` to stretch in perpendicular direction:
![image](https://github.com/cdelker/schemdraw/assets/9392655/b2d1a252-9904-48f2-9b75-a78232f34de6)

**Comment by pozitron57** [comment]
> Added in [40311ed](https://github.com/cdelker/schemdraw/commit/40311edb6be70cd9ae8886e70627db23edb660f2).

Thanks! I tested it a bit. As far as I understand, if scale, scalex and scaley are applied together,
only the last option is used. I think it is sane when scale is mixed with scalex and/or scaley,
but `.scalex(3).scaley(2)` can work together just fine.

**Comment by cdelker** [comment]
Yeah, one function overwrote the other. f507534cf95f0da1f051a2a87e65d37aee447f7c should fix it so both may be applied at the same time.
